qipr_approver/approver/views/index.py

```diff
@@ -5,12 +5,12 @@
 from django.contrib.auth.models import User
 from approver.models import Project
 
-import json
-import approver.utils as utils
 import approver.constants as constants
+import approver.utils as utils
+from approver.views.dashboard import dashboard
 
 def index(request):
-    context = {
-        'content': 'approver/index.html',
-    }
-    return utils.layout_render(request, context)
+    if(utils.get_current_user_gatorlink(request.session)):
+        return dashboard(request)
```

Comment:
Please check this to make sure that you cannot manually add a session variable to the request with some random gatorlink and get to a dashboard page. _If_ it is possible I think the worst that will happen is an error page, but I'd like you to check if you haven't.